content/S2016/S16-event-office-move.md

```diff
@@ -0,0 +1,11 @@
+Title: Our office has moved!
+Date: 2016-8-15
+Category: Events
```

Comment:
do we need to use this category? do things show up on the front page if make it a different category? I feel like it shouldn't be an event and more of an announcement thing